pages/common/supervisord.md

```diff
@@ -0,0 +1,13 @@
+# supervisord
+
+> Supervisor is a client/server system that allows its users to control a number of processes on UNIX-like operating systems.
+> Supervisord is the server piece of supervisor.
+> Most of supervisord's config should be placed in configuration file, some basic usage will be listed here.
```

Comment:
Please reduce this description to two lines. You could merge the second and third lines into a single line, and separate them with a semicolon, for example.